integration-cli/docker_api_containers_test.go

```diff
@@ -771,6 +771,44 @@ func UtilCreateNetworkMode(c *check.C, networkMode string) {
 	}
 }
 
+func (s *DockerSuite) TestContainerApiCreateWithCpuSharesCpuset(c *check.C) {
+
+	config := map[string]interface{}{
+		"Image":      "busybox",
+		"CpuShares":  512,
+		"CpusetCpus": "0,1",
+	}
+
+	status, body, err := sockRequest("POST", "/containers/create", config)
+	c.Assert(err, check.IsNil)
+	c.Assert(status, check.Equals, http.StatusCreated)
+
+	var container types.ContainerCreateResponse
+	if err := json.Unmarshal(body, &container); err != nil {
+		c.Fatal(err)
+	}
+
+	status, body, err = sockRequest("GET", "/containers/"+container.ID+"/json", nil)
+	c.Assert(err, check.IsNil)
+	c.Assert(status, check.Equals, http.StatusOK)
+
+	var containerJson types.ContainerJSON
+	if err := json.Unmarshal(body, &containerJson); err != nil {
+		c.Fatal(err)
+	}
+
+	out, err := inspectField(containerJson.Id, "HostConfig.CpuShares")
+	c.Assert(out, check.Equals, "512")
+	if err != nil {
+		c.Fatal(err, out)
+	}
+	outCpuset, errCpuset := inspectField(containerJson.Id, "HostConfig.CpusetCpus")
+	c.Assert(outCpuset, check.Equals, "0,1")
+	if err != nil {
```

Comment:
this condition will evaluate the previous error not the `errCpuset`, so pls change `errCpuset` to just `err`